content/vault/v1.20.x/content/docs/deploy/kubernetes/vso/helm.mdx

```diff
@@ -39,6 +40,8 @@ Use these links to navigate to a particular top-level stanza.
 
   - `replicas` ((#v-controller-replicas)) (`integer: 1`) - Set the number of replicas for the operator.
 
+  - `priorityClassName` ((#v-controller-priorityclassname)) (`string: ""`) - Set the priority class for the operator.
```

Comment:
`priorityClassName`, `topologySpreadConstraints`, and `podDisruptionBudget` are not part of this work, but they got auto-generated along with the rest of this because it looks like docs generation got missed last time.

Reply:
The Helm docs are generated by the `make gen-helm-docs` command in the [vault-secrets-operator](https://github.com/hashicorp/vault-secrets-operator) repo. (Well, this particular iteration is from a private fork of that repo, but that's typically where the generation is done.)

Is the Education team typically added as a codeowner for field descriptions in the main Vault repo, like wherever the Vault CLI help text lives, for example? I would say this is similar to that -- the text lives with the code itself, but we have a helper script that spits it out in this format.